### PR TITLE
Decapitalization merge

### PR DESCRIPTION
**Pull request by Snaid1**

### CODE REVIEW

File: data/maps/CeladonCity_Gym/text.inc [hunk 1]
```diff
@@ -19,37 +19,37 @@ CeladonCity_Gym_Text_ErikaDefeat::
     .string "I concede defeat.\l"
     .string "You are remarkably strong.\p"
     .string "I must confer on you the\n"
-    .string "RAINBOWBADGE.$"
```
Comment (Thill020) [inline data/maps/CeladonCity_Gym/text.inc:22]: Do we want to leave this as 1 word, or break it up into Rainbow Badge? The item name is never seen anywhere, and in reality they are two words

File: data/maps/PalletTown_ProfessorOaksLab/scripts.inc [hunk 2]
```diff
@@ -59,7 +59,7 @@ PalletTown_ProfessorOaksLab_OnFrame::
 	map_script_2 VAR_MAP_SCENE_PALLET_TOWN_PROFESSOR_OAKS_LAB, 7, PalletTown_ProfessorOaksLab_EventScript_EnterForNationalDexScene
 	.2byte 0
 
-PalletTown_ProfessorOaksLab_EventScript_EnterForNationalDexScene:: @ 8169002 
+PalletTown_ProfessorOaksLab_EventScript_EnterForNationalDexScene::
```
Comment (Thill020) [inline data/maps/PalletTown_ProfessorOaksLab/scripts.inc]: Here you removed a reference call I believe. This was not a Decapitalization.

File: data/maps/Route15/text.inc [hunk 3]
```diff
@@ -117,38 +117,38 @@ Route15_Text_YazminPostBattle::
...
 
 Route15_Text_MyaNotEnoughMons::
-    .string "MYA: Do you want to challenge us?\n"
-    .string "You'll need two POKéMON, though.$"
+    .string "Mya: Do you want to challenge us?\n"
+    .string "You'll need two Pokémon, though.$"
 
-Route15_Text_RonIntro::
-    .string "RON: My sister gets scary when we\n"
+Route15_Text_RonIntro::
```
Comment (Thill020) [inline data/maps/Route15/text.inc]: I don't know if it's an issue or not, but there's an extra space here.

File: src/data/text/teachy_tv.h [hunk 4]
```diff
@@ -88,26 +88,26 @@ const u8 gTeachyTvText_MatchupsScript1[] = _(
...
+    "a threat to your Pokémon?\p"
     "Is there any chance that it may\n"
     "have disastrously tough moves?\p"
     "Watch the type matchups to gain\n"
     "the upper hand!\p"
     "All righty, be seeing you!\p"
-    "Oh, for the COOL-type POKé DUDE,\n"
-    "AWESOME-type kids like you match\l"
+    "Oh, for the Cool-type Poké Dude,\n"
+    "Awesome-type kids like you match\l"
```
Comment (Thill020) [inline src/data/text/teachy_tv.h:103]: the \l looks off, from everything I've seen so far, I wanna say it should be \$

**Review by Thill020 — COMMENTED** [review]
All files have been looked through, everything outside of my notes look good to the best of my knowledge.

File: src/data/text/teachy_tv.h (hunk 4 above)
Comment (Snaid1) [reply src/data/text/teachy_tv.h:103]: That's how it is in the vanilla branch. no changes shown so I'm not fixing this one.

**Pull request merged**